### ISSUE TITLE
How frequently can a users FLoCk change?

### ISSUE THREAD

**Issue by brodrigu** [jkarlin/floc#22]
It's possible that a users browsing habits will change over time, possibly moving them between cohorts. How frequently will Google allow the users FLoCk to change?

**Comment by brodrigu** [comment]
@jkarlin It appears the codebase is using 1 day as the current working expiration of a floc. 

https://github.com/chromium/chromium/blob/master/chrome/browser/federated_learning/floc_id_provider_impl.cc#L29

Can you confirm I am reading the code correctly here?

**Comment by jkarlin** [comment]
You're correct that we're currently exploring 24 hour updates. But please don't read too much into the current code. We're taking baby steps.

**Comment by brodrigu** [comment]
Understood, tuning the ttl to provide the right balance of privacy and usefulness when exposing different IDS will be tricky as you noted in #20.

Thanks!
